test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -103,19 +105,19 @@ import           Test.Pos.Block.Logic.Emulation (Emulation (..), runEmulation, s
 -- | This data type contains all parameters which should be generated
 -- before testing starts.
 data TestParams = TestParams
-    { _tpGenTxpContext     :: !GenesisTxpContext
-    -- ^ Genesis 'Utxo'.
-    , _tpAllSecrets        :: !AllSecrets
+    { _tpGenTxpContext      :: !GenesisTxpContext
+    -- ^ Genesis txp-related data.
+    , _tpAllSecrets         :: !AllSecrets
     -- ^ Secret keys corresponding to 'PubKeyAddress'es from
     -- genesis 'Utxo'.
     -- They are stored in map (with 'StakeholderId' as key) to make it easy
     -- to find 'SecretKey' corresponding to given 'StakeholderId'.
     -- In tests we often want to have inverse of 'hash' and 'toPublic'.
-    , _tpStakeDistribution :: !StakeDistribution
-    -- ^ Stake distribution which was used to generate genesis utxo.
-    -- It's primarily needed to see which distribution was used (e. g.
+    , _tpStakeDistributions :: ![StakeDistribution]
```

Comment:
Because in `Test.Pos.Lrc.WorkerSpec.genTestParams` I use multiple distributions. And it was convenient for debugging.